scale/job/models.py

```diff
@@ -1030,6 +1031,10 @@ def schedule_job_executions(self, framework_id, job_executions, workspaces):
             if resources is None:
                 raise Exception('Cannot schedule job execution %i without resources' % job_exe.id)
 
+            # Populate job_configuration with default settings
+            configuration = job_exe.get_job_configuration()
+            configuration.populate_default_job_settings(job_exe)
```

Comment:
This doesn't get set back to the job_exe.configuration field. Move this to configure_docker_params(), somewhere near the top is fine.